apriltags2_ros/CMakeLists.txt

```diff
@@ -116,3 +116,16 @@ target_link_libraries(${PROJECT_NAME}_single_image_client_node
   common
   ${catkin_LIBRARIES}
 )
+
+foreach(dir launch config)
+  install(DIRECTORY ${dir}/
```

Comment:
Can be simplified to `install(DIRECTORY dir launch config DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}`

Reply:
I will make this change.